src/main/drivers/accgyro/accgyro_spi_bmi160.c

```diff
@@ -366,9 +325,9 @@ bool bmi160AccRead(accDev_t *acc)
     uint8_t bmi160_rec_buf[BUFFER_SIZE];
     uint8_t bmi160_tx_buf[BUFFER_SIZE] = {BMI160_REG_ACC_DATA_X_LSB | 0x80, 0, 0, 0, 0, 0, 0};
 
-    ENABLE_BMI160(acc->bus.spi.csnPin);
-    spiTransfer(BMI160_SPI_INSTANCE, bmi160_rec_buf, bmi160_tx_buf, BUFFER_SIZE);   // receive response
-    DISABLE_BMI160(acc->bus.spi.csnPin);
+    IOLo(acc->bus.spi.csnPin);
```

Comment:
This is actually `spiReadRegisterBuffer(bus, BMI160_REG_ACC_DATA_X_LSB, 6, bmi160_rec_buf);`, when IDX_REG is removed from `bmi160_rec_buf` / enum

BMI ignores SDI oon read command, so only difference is that SDI will be high instead of low